aptos-move/e2e-move-tests/src/harness.rs

```diff
@@ -154,45 +154,36 @@ impl MoveHarness {
         }
     }
 
+    pub fn store_and_fund_account(&mut self, acc: &Account, balance: u64, seq_num: u64) -> Account {
```

Comment:
nit: I think you can pass the value, not the reference?

Reply:
I cannot because of the use here - https://github.com/aptos-labs/aptos-core/pull/10758/files#diff-0f8c14d15ff465f5c7c9a1b9ad05ade378347ccacddecb9f67771526407acb88

where I fund the same account in multiple environments